### ISSUE TITLE
512x512 training  configuration

### ISSUE THREAD

**Issue by why529913** [KaiChen1998/GeoDiffusion#8]
I want to know the specific configuration of 512x512 size training, --num_bucket_per_side should be 512, 512?

**Comment by KaiChen1998** [comment]
The `num_bucket_per_side` for nuImages is `512x512`, while `256x256` for COCO-Stuff. Check the generation configs in our pre-trained checkpoints for reference [here](https://github.com/KaiChen1998/GeoDiffusion?tab=readme-ov-file#download-pre-trained-models). 

Other configurations are maintained similarly to the `256x256` experiments. The only thing to change is to linearly scale the learning rate with respect to the batch size.
